.github/policies/IssueManagement.CloseResolutions.yml

```diff
@@ -0,0 +1,130 @@
+id: CloseResolutionTags
+name: GitOps.PullRequestIssueManagement
+description: Closing issues with Resolution* 
+owner:
+resource: repository
+disabled: false
+where:
+configuration:
+  resourceManagementConfiguration:
+    scheduledSearches:
+    - description: Close if marked as Resolution-Declined after one day of no activity
```

Comment:
I would be more readable if you can insert 1 empty line before each `- description: xxxx` (except for the 1st one), so it's easy to see where one ends and where one starts.